src/tools/wasm-merge.cpp

```diff
@@ -296,12 +296,13 @@ struct InputMergeable : public ExpressionStackWalker<InputMergeable, Visitor<Inp
       return;
     }
     curr->name = gNames[curr->name];
-    assert(curr->name.is());
-    // if this is the memory or table base, add the bump
-    if (memoryBaseGlobals.count(curr->name)) {
-      addBump(outputMergeable.totalMemorySize);
-    } else if (tableBaseGlobals.count(curr->name)) {
-      addBump(outputMergeable.totalTableSize);
+    if (curr->name.is()) {
```

Comment:
In what case is this not valid? Re-reading the code, gNames is the mapping of old to new names, so why wouldn't a name have a mapping entry?